src/app/asset-filters/asset-filters.component.ts

```diff
@@ -198,7 +198,7 @@ export class AssetFilters {
 
         // The loggedd in user's institutional 'Collection Type' filter
         if (colType.name == '2' || colType.name == '4') {
-          filters['collectiontypes'][i]['count'] = this.instFilterCount
```

Comment:
`instFilterCount` is explicitly set for this purpose because it should reference the `donatinginstitutionids` count

Reply:
Sorry I didn't fully understand the requirement, now it's back. The purpose now is to remove the filter header if it only contains one facet and the name of the facet is empty or the count is 0. New changes have been made~